### PR TITLE
change the verbose effect

### PR DESCRIPTION
**Pull request by iLoveBug**
I set the verbose default as "false", and keep the generate function always print the tokens. the verbose argument only enable the output of prompt and timing information.

### CODE REVIEW

**Comment by awni** [comment]
I'm curious, why do you want verbose `False` with the CLI? I'm not against it just wondering why it's important?

File: llms/mlx_lm/utils.py [hunk 1]
```diff
@@ -157,19 +158,19 @@ def generate(
...
+        s = tokenizer.decode(tokens)
+        if REPLACEMENT_CHAR not in s:
             if formatter:
                 formatter(s[skip:], prob.item())
-                skip = len(s)
-            elif REPLACEMENT_CHAR not in s:
+            else:
                 print(s[skip:], end="", flush=True)
-                skip = len(s)
+            skip = len(s)
```
Comment (awni) [inline llms/mlx_lm/utils.py:167]: The formatter expects one token / prob, so this will break in some cases.

File: llms/mlx_lm/utils.py [hunk 2]
```diff
@@ -157,19 +158,19 @@ def generate(
...
 
     tokens = tokenizer.decode(tokens).replace(REPLACEMENT_CHAR, "")
 
+    print(tokens[skip:], flush=True)
```
Comment (awni) [inline llms/mlx_lm/utils.py:171]: We only want to print this in verbose mode.

File: llms/mlx_lm/utils.py (hunk 1 above)
Comment (awni) [inline llms/mlx_lm/utils.py:167]: We don't want to print here in general. The API shouldn't print anything when `verbose=False`

**Comment by iLoveBug** [comment]
OK，I understood. Thanks for your review.

**Pull request closed without merging**